Tests/MutationOperators/RemoveSideEffectsOperatorSpec.swift

```diff
@@ -75,6 +75,21 @@ class RemoveSideEffectsOperatorSpec: QuickSpec {
 
                 expect(visitor.positionsOfToken.first?.line).to(equal(3))
             }
+            
+            it("ignores functions that are used as implicit return") {
+                let sampleWithImplicitReturn = sourceCode(fromFileAt: "\(self.mutationExamplesDirectory)/SideEffect/sampleWithImplicitReturn.swift")!
+                let visitor = RemoveSideEffectsOperator.Visitor(sourceFileInfo: sampleWithImplicitReturn.asSourceFileInfo)
+                visitor.walk(sampleWithImplicitReturn.code)
+
+                guard visitor.positionsOfToken.count == 3 else {
+                    fail("Expected 3 token to be discovered, got \(visitor.positionsOfToken.count) instead")
+                    return
+                }
+
+                expect(visitor.positionsOfToken[0].line).to(equal(2))
```

Comment:
I don't know how much we care for this PR specifically, but Quick/Nimble (I can never remember which one) has a nice shorthand:

```swift
expect(foo) == bar
```